tests/test_browser.py

```diff
@@ -7,10 +7,16 @@
 import pytest
 
 
-def test_submit_online():
+@pytest.fixture
+def httpbin():
+    from utils import HttpbinWrapper
+    return HttpbinWrapper()
```

Comment:
Ideally the 3 instances of this would be factored into a common file. If I read correctly https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions, moving it to a `conftest.py` file should do it.

But since this is meant to be temporary anyway, I don't have strong objection in keeping it as-is.

Reply:
Oh, and could this become a conditional code that would actually do the pytest-httpbin thing when it works, and fall back to HttpbinWrapper if not?